CHANGELOG.md

```diff
@@ -1,5 +1,14 @@
 # Changelog
 
+## [1.2.3](https://github.com/Pinterest/PINOperation/tree/1.2.3) (TBD)
```

Comment:
I can update the version number and release date before merging, per guidance from the maintainers.